plugin/hover.py

```diff
@@ -174,21 +175,27 @@ def _show_hover(self, point: int) -> None:
                     location=point,
                     wrapper_class=css().popups_classname,
                     max_width=800,
-                    on_navigate=lambda href: self.on_hover_navigate(href, point))
+                    on_navigate=lambda href: self._on_navigate(href, point))
 
-    def on_hover_navigate(self, href: str, point: int) -> None:
-        if href.startswith('subl:'):
+    def _on_navigate(self, href: str, point: int) -> None:
+        if href.startswith("subl:"):
             pass
-        elif href.startswith('code-actions'):
+        elif href.startswith('code-actions:'):
             _, config_name = href.split(":")
             titles = [command["title"] for command in self._actions_by_config[config_name]]
             sel = self.view.sel()
             sel.clear()
             sel.add(sublime.Region(point, point))
-
             self.view.show_popup_menu(titles, lambda i: self.handle_code_action_select(config_name, i))
+        elif href.startswith("file://"):
```

Comment:
Would prefer `location://` as it's not actually a valid `file://` URI.

Reply:
c48dd48